src/data.js

```diff
@@ -0,0 +1,382 @@
+/* Data
+/  
+/  Interpret API data fetched by query.js
+/  
+/  Direct link to render() functions 
+*/
+
+// Interprete and render project using parentProjectId, id and name
+class data {
```

Comment:
If the name of this class was something like ApiHandler you wouldn't need the comment in the top